src/pages/RestrictedAction/Workspace/WorkspaceOwnerRestrictedAction.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import {View} from 'react-native';
+import Badge from '@components/Badge';
+import Button from '@components/Button';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import * as Illustrations from '@components/Icon/Illustrations';
+import ScreenWrapper from '@components/ScreenWrapper';
+import ScrollView from '@components/ScrollView';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import ROUTES from '@src/ROUTES';
+
+function WorkspaceOwnerRestrictedAction() {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+
+    const addPaymentCard = () => {
+        Navigation.navigate(ROUTES.SETTINGS_SUBSCRIPTION);
+    }
+
+    return (
+        <ScreenWrapper
+            includeSafeAreaPaddingBottom
+            testID={WorkspaceOwnerRestrictedAction.displayName}
+        >
+            <HeaderWithBackButton
+                title={translate('workspace.restrictedAction.restricted')}
+                onBackButtonPress={Navigation.goBack}
+            />
+            <ScrollView>
```

Comment:
We're missing the 12px of top padding below the page header and above the card:
![CleanShot 2024-06-19 at 09 52 42@2x](https://github.com/Expensify/App/assets/2319350/b3586857-da49-4228-b184-a5ce1b590ef7)

From Figma:
![CleanShot 2024-06-19 at 09 53 50@2x](https://github.com/Expensify/App/assets/2319350/705927cf-ae2f-4fce-bd60-c801dbfbce2b)